nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -163,7 +163,11 @@ def forward(self, x, pad_mask=None):
             x.masked_fill_(pad_mask.unsqueeze(1), 0.0)
 
         x = self.depthwise_conv(x)
-        x = self.batch_norm(x)
+        if torch.onnx.is_in_onnx_export():
+            with torch.cuda.amp.autocast(enabled=False):
```

Comment:
I thought BN was the primary focus of autocast. Does it fail for some reason under onnx context?

Reply:
I am putting it under autocast(False) because otherwise it generates bad ONNX code